src/components/layout/index.tsx

```diff
@@ -46,7 +45,9 @@ export function getLayout(layoutProps: LayoutProps, lastGenerated: string) {
 
 export default Layout;
 
-function Layout(props: WithChildren<LayoutProps & ILastGeneratedData>) {
+function Layout(
+  props: LayoutProps & ILastGeneratedData & { children: React.ReactNode }
```

Comment:
Here you can integrate things with the already existing layout props

```suggestion
export interface LayoutProps extends ILastGeneratedData {
  url?: string;
  title: string;
  description?: string;
  openGraphImage?: string;
  twitterImage?: string;
  children: React.ReactNode;
}
```